_includes/releases/v22.2/v22.2.0-alpha.3.md

```diff
@@ -0,0 +1,99 @@
+- We now support `DISCARD {TEMP,TEMPORARY}`, which drops all [temporary tables](../v22.2/temporary-tables.html) created in the current session. The command does not drop temporary schemas. [#86246][#86246]
+- [`CREATE SCHEDULE`](../v22.2/create-schedule-for-backup.html) is no longer an admin only operation. Users should grant the appropriate [`BACKUP`](../v22.2/backup.html) privileges on the targets they wish to back up as part of the schedule. Cluster [backups](../v22.2/backup.html) require admin or system privilege `BACKUP`; DB backups require database privilege `BACKUP`; table backups require table privilege `BACKUP`. Release justification: high impact change for fine grained permission control. [#87188][#87188]
+- Decreased the cardinality of the number on `__moreN__` when replacing literals. [#87202][#87202]
+- When adding a `SECONDARY REGION` to a [multi-region database](../v22.2/multiregion-overview.html), the region is implicitly added to the regions list of the database if it was not present already. [#87108][#87108]
+- Added the `pg_get_function_def` [function](../v22.2/functions-and-operators.html), which returns the `CREATE` statement that can be used to create the given user-defined function. For built-in functions, it only returns the name of the function. Release justification: low risk enhancement for compatibility. [#87439][#87439]
+
+<h3 id="v22-2-0-alpha-3-operational-changes">Operational changes</h3>
+
+- This change introduces a new histogram implementation that will reduce the total number of buckets and standardize them across all usage. This should help increase the usability of histograms when exported to a UI (i.e., Grafana) and reduce the storage overhead. After applying this patch it is expected to see fewer buckets in prometheus/grafana, but still  have similar values for histogram percentiles due to the use of interpolated values by Prometheus. [#86671][#86671]
+
+<h3 id="v22-2-0-alpha-3-command-line-changes">Command-line changes</h3>
+
+- Improved the output of sinkless [changefeeds](../v22.2/create-changefeed.html) in the [`cockroach sql`](../v22.2/cockroach-sql.html) terminal. Release justification: Prevents UX degradation in v22.2 as a side effect of fixing the `bytea_output` bug. [#85181][#85181]
+
+<h3 id="v22-2-0-alpha-3-db-console-changes">DB Console changes</h3>
+
+- In the [session details](../v22.2/ui-sessions-page.html) page, users can click on a transaction fingerprint id from the list of cached transaction fingerprints to go to that transaction's details page. The app will also change the selected date range to that of the session's start (rounded down to the hour) and end time (rounded up to the hour) on click. [#86919][#86919]
```

Comment:
Maybe cap on "session details" as it's a console page?

Reply:
And the other page names in this section. I think this is what we tend to do across the docs.

Reply:
Capitalized Session Details

Also capitalized other page names I found.  Sorry, guess I don't know the UI well enough!